### ISSUE TITLE
Hørsel

### ISSUE THREAD

**Issue by anjamaa** [realfagstermer/realfagstermer#989]
Legge til `Hørselssans` på norsk?

--
URI: http://data.ub.uio.no/realfagstermer/c010461
Bruk: http://ub-viz01.uio.no/okapi2/#/search?q=realfagstermer%3A%22H%C3%B8rsel%22
Soksed: https://soksed.biblionaut.net/concepts/REAL010461

**Comment by Hypsibius** [comment]
Hadde vi ikkje ein diskusjon om sansebiologi før? Kva konkluderte vi med?

**Comment by Hypsibius** [comment]
Jo, så sanneleg: https://github.com/realfagstermer/realfagstermer/issues/640

**Comment by danmichaelo** [comment]
Mm, vi konkluderte vel ikke helt
